birdhouse/config/jupyterhub/config/magpie/docker-compose-extra.yml

```diff
@@ -1,7 +1,5 @@
 version: "3.4"
 services:
   magpie:
-    environment:
-      MAGPIE_USER_NAME_EXTRA_REGEX: ^\S+$$
```

Comment:
I assume you migrate this change to `magpie.ini` to ensure forward compat if later JupyterHub is enabled?  Missing corresponding change in `magpie.ini`.  Please use an explicit regex `"^[a-zA-Z0-9_\-\.]+$$"` which would allow upper case, `_-.` and nothing else.

This is to ensure even if Magpie validation regex changes later, we are independent and as @fmigneault said, this is to simplify the life of other node admin, avoiding them to lookup the validation regex on Magpie side.

Reply:
When adding back the extra regex, please add a comment that Magpie already has a built-in regex too, just to raise awareness for those not in the know.

I guess now that Magpie do not allow uppercase, the extra regex becomes `"[a-z0-9_\-\.]+$$"`.

Reply:
@tlvu 

With the new changes, we do not need to use the `MAGPIE_USER_NAME_EXTRA_REGEX` feature at all. The current default checks that Magpie does are sufficient (without the need to do an extra check), see https://github.com/Ouranosinc/Magpie/pull/597 for details

Reply:
@mishaschwartz 

I see, thanks.  In this case, just to help other future node admin not in the know, can you document in `magpie.ini` the exact permanent link to the magie regex, this one currently https://github.com/Ouranosinc/Magpie/blob/595602d2cebae94223b952f0cf04a1caa64f6546/magpie/api/management/user/user_utils.py#L66. 

So this is the regex we expect at this moment.  If in the future this regex changes on Magpie side, and somehow break us, we will know what was the original expected working regex.